package.json

```diff
@@ -8,8 +8,7 @@
     "build": "next build",
     "test": "jest",
     "start": "NODE_ENV=production node server.js",
-    "format-staged": "pretty-quick --staged --no-restage --bail",
-    "format-check": "pretty-quick --check --verbose",
+    "format": "prettier --write \"**/*.+(js|md|json)\"",
```

Comment:
- OK to merge those into a single `format` script, but it be used at CI, so it should not `--write`, just check and fail if it finds a problem. (Equivalent at `--check --verbose` for the former `format-check": "pretty-quick` script. Thanks.
- Please use `(js|json|md)` (alphabetical order).

Reply:
Actually... I'm not sure we can remove `format-check`. Let me think...

Reply:
Yes, never mind. It's OK, we just need to use `format` in the CI config now. See https://github.com/iterative/dvc.org/pull/790#issuecomment-554034488 🙂

Reply:
I addressed this in my recent merge from master.

Reply:
Nevermind, part of this still requires addressing.

Reply:
the `format` script is intended to format the whole codebase, I think if we want to check the formatting it's better to rename the script to `check-format` instead of just `format`.
So I suggest to rename the `format` script to `check-format` and run prettier with the parameters you've mentioned.

Reply:
Hmmm good point. But we also have the `lint` script, what would we rename it to? `check-lint`? That doesn't sound as nice as `check-format`. I think just go back to `format-check` and `lint-check` respectively, is best, please.

Reply:
p.s. Actually, we don't intend to have scripts to actually format the code. We have information on how to do that in https://dvc.org/doc/user-guide/contributing/docs#doc-style-guidelines-and-tips-for-java-script-and-markdown.

Reply:
I don't see there is a reason to rename the `lint` script since its name gives the exact intent of it, I'll only rename the `format` script to `check-format` and replace the `--write` arg with `--check`

> p.s. Actually, we don't intend to have scripts to actually format the code. We have information on how to do that in https://dvc.org/doc/user-guide/contributing/docs#doc-style-guidelines-and-tips-for-java-script-and-markdown.

Good Point!